provider/fs/README.md

```diff
@@ -11,7 +11,7 @@ the filesystem via an iterable data provider (see the `export` module).
 use icu_provider_fs::FsDataProvider;
 
 let provider = FsDataProvider::try_new("/path/to/data/directory")
-    .expect_err("Specify a real directory in the line above");
+    .expect("Specify a real directory in the line above");
```

Comment:
Issue: The test is written with `expect_err` because it needs to pass `cargo test --doc`.  You could

1. Disable the code from running as a test (this will cause it to show up orange in the generated docs)
2. Remove the assertion
3. Leave it alone

I think I prefer (3) but I can see reasons why (1) or (2) may be desirable.  At the end of the day, this is a doc string, so it should be what is most helpful to ICU4X consumers.

Reply:
That's fair -- I think I'll drop that commit.